pkg/sign/notary.go

```diff
@@ -57,10 +59,13 @@ type AuthSecretConfig struct {
 
 // SignifySecret contains configuration of secret that is used to connect to SAP signify service
 type SignifySecret struct {
-	// URL to the SAP signify endpoint
-	Endpoint string `yaml:"endpoint" json:"endpoint"`
-	// Actual secret data that will be pushed on sign request. Assumed its in JSON format
-	Payload string `yaml:"payload" json:"payload"`
+	CreatedAt      string `json:"createdAt"` //TODO waiting for Kacper to provide information
+	TokenURL       string `json:"tokenURL"`
+	CertServiceURL string `json:"certServiceURL"`
+	ClientID       string `json:"clientID"`
+	CertficateData string `json:"certData"`
+	PrivateKeyData string `json:"privateKeyData"`
+	KeyPassword    string `json:"password"`
```

Comment:
Image signing proces does not need all this data. The signer needs only a `tls.X509KeyPair` which could be prepared by rotation service. The signer would just read the KeyPair file pulled from secret manager.

Reply:
Reduced to cert and pk.